src/components/TeachingBubble/TeachingBubble.Props.ts

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TeachingBubble } from './TeachingBubble';
+
+/**
+ * Checkbox class interface.
+ */
+export interface ITeachingBubble {
+
+}
+
+/**
+ * TeachingBubble component props.
+ */
+export interface ITeachingBubbleProps extends React.Props<TeachingBubble> {
+  /**
+   * The background color of the TeachingBubble
+   * @default white
+   */
+  backgroundColor?: TeachingBubbleBGColor;
```

Comment:
If you want to limit the background colors, then instead using a semantic enum like TeachingBubbleType so that customers can pick the right type (and thus, the right color).